<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BurningControlWindow</class>
 <widget class="QGroupBox" name="BurningControlWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>326</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>812</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox_Image">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>82</height>
      </size>
     </property>
     <property name="title">
      <string>Image</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="4" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_Open">
        <item>
         <widget class="QLineEdit" name="inputSysImage"/>
        </item>
        <item>
         <widget class="QPushButton" name="btnSelectImage">
          <property name="text">
           <string>Open</string>
          </property>
          <property name="icon">
           <iconset resource="../../resources/main.qrc">
            <normaloff>:/folder-open-solid.svg</normaloff>:/folder-open-solid.svg</iconset>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <widget class="QTableView" name="tableView">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>16777215</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QGroupBox" name="groupBox_Util">
     <property name="title">
      <string/>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="7">
       <widget class="QPushButton" name="btnStartBurn">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>42</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>11</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Start</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/main.qrc">
          <normaloff>:/file-arrow-down-solid.svg</normaloff>:/file-arrow-down-solid.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>75</width>
          <height>20</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QComboBox" name="inputTarget"/>
      </item>
      <item row="0" column="4">
       <widget class="QPushButton" name="btnOpenSettings">
        <property name="minimumSize">
         <size>
          <width>140</width>
          <height>0</height>
         </size>
        </property>
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="text">
         <string>Setting</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/main.qrc">
          <normaloff>:/sliders-solid.svg</normaloff>:/sliders-solid.svg</iconset>
        </property>
        <property name="flat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="5">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="1">
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="text">
         <string>Medium</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="AnimatedButton" name="buttonStartAuto">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>130</width>
          <height>42</height>
         </size>
        </property>
        <property name="text">
         <string>Auto</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources/main.qrc">
          <normaloff>:/robot-solid.svg</normaloff>:/robot-solid.svg</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>20</width>
          <height>20</height>
         </size>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AnimatedButton</class>
   <extends>QPushButton</extends>
   <header>widgets/AnimatedButton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/main.qrc"/>
 </resources>
 <connections/>
</ui>
